### ISSUE TITLE
bare + browser :) I think you underestimate how bare my environment is

### ISSUE THREAD

**Issue by lawless-m** [duckdb/duckdb-wasm#443]
I'm trying to get DuckDB-WASM going on my own site

without being steeped in Bundling lore (I do write my own and use D3 no not entirely naked) I'm hitting a brick wall.

If one serves the files from 

duckdb-wasm/examples/bare-browser/

then all you end up with is 

Loading failed for the module with source “http://localhost/duckdb-browser.mjs”.

**Comment by ankoh** [comment]
I'm sorry our examples are tested within the yarn workspace.
I understand that this is unnecessarily complex, we need to make our examples independent of the monorepo to allow for a simple c&p.

**Comment by lawless-m** [comment]
the cliff is quite steep

/me harks back to the days to ./configure && make && sudo make install

**Comment by braaannigan** [comment]
Hi @ankoh - I'm very excited about your project. I'm not a front-end expert, but I've managed to get the bare-browser example working with yarn.

However, when I inspect the console I see an output that shows the query, but not an output that shows the output of the query. Can you let me know how I can generate that?

Also, would you be interested in a PR that adds a readme to the browser examples explaining how to run them?

**Comment by gcallxpert** [comment]
A vanilla JavaScript example would be great. (not using any bundlers)

Spent a day and still couldn't have one up and running

**Comment by okpatil4u** [comment]
I agree with @gcallxpert. It would be easier to understand and use it this way.

**Comment by sgilligan** [comment]
If this helps - I've created a sandbox demo - https://codesandbox.io/s/7ss39r?file=/src/DuckDb.js

**Comment by okpatil4u** [comment]
Thank you @sgilligan, it helps !!
